trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/generate_airfoil_transitions.py

```diff
@@ -0,0 +1,100 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
+# compute_airfoil_transition.py
+# 
+# Created:  Mar 2021, R. Erhard
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.import_airfoil_geometry import import_airfoil_geometry 
+from SUAVE.Plots.Geometry_Plots import plot_airfoil
+import numpy as np
+
+def generate_airfoil_transition(a1, a2, space, nairfoils, save_file=False, save_filename="Transition", npts=50):
+    """ Takes in two airfoils and their locations and interpolates
+    between them to establish a transition region.
```

Comment:
can you find a better word than transition? When I see airfoil transition I just think laminar/turbulent.

maybe just airfoil_interpolation

Reply:
changed to: generate_interpolated_airfoils